modules/nw-ovn-ipsec-verification.adoc

```diff
@@ -14,26 +14,22 @@ As a cluster administrator, you can verify that IPsec is enabled.
 +
 [source,terminal]
 ----
-$ oc get pods -n openshift-ovn-kubernetes | grep ovnkube-master
+$ oc get pods -n openshift-ovn-kubernetes | grep ovnkube-control-plane
 ----
 +
 .Example output
 [source,terminal]
 ----
-ovnkube-master-4496s        1/1     Running   0          6h39m
-ovnkube-master-d6cht        1/1     Running   0          6h42m
-ovnkube-master-skblc        1/1     Running   0          6h51m
-ovnkube-master-vf8rf        1/1     Running   0          6h51m
-ovnkube-master-w7hjr        1/1     Running   0          6h51m
-ovnkube-master-zsk7x        1/1     Running   0          6h42m
+ovnkube-control-plane-66564dbd96-r8vlp   2/2     Running   0             34m
+ovnkube-control-plane-66564dbd96-rv5lw   2/2     Running   0             34m
+ovnkube-control-plane-66564dbd96-wvkgf   2/2     Running   2 (25m ago)   34m
 ----
 
 . Verify that IPsec is enabled on your cluster:
 +
 [source,terminal]
 ----
-$ oc -n openshift-ovn-kubernetes -c nbdb rsh ovnkube-master-<XXXXX> \
-  ovn-nbctl --no-leader-only get nb_global . ipsec
+$ oc -n openshift-ovn-kubernetes -c nbdb rsh ovnkube-master-<XXXXX> ovn-nbctl --no-leader-only get nb_global . ipsec
```

Comment:
please note that there is no ovnkube-master on 4.14 @JoeAldinger 
its ovnkube-node pod and that also tells me we need to be showing the ovnkube-node pods in the output?